src/plot_api/plot_schema.js

```diff
@@ -486,7 +486,7 @@ function getLayoutAttributes() {
 
         if(!_module.layoutAttributes) continue;
 
-        if(_module.name === 'cartesian') {
+        if(_module.name === 'cartesian' || _module.name === 'splom') {
```

Comment:
Can we do this just based on `_module.attr`?
`if(Array.isArray(_module.attr)) { _module.attr.forEach(... handleBasePlotModule) }`